src/cli.js

```diff
@@ -228,8 +214,70 @@ program
       const answers = await inquirer.prompt(questions)
       await tasks.quickstart(api, answers, space)
     } catch (e) {
-      console.log(chalk.red('X') + ' An error ocurred when execute quickstart operations')
-      console.error(e)
+      console.log(chalk.red('X') + ' An error ocurred when execute quickstart operations: ' + e.message)
+      process.exit(1)
+    }
+  })
+
+program
+  .command('generate-migration')
+  .description('Generate a migration file to Storyblok components schema')
+  .requiredOption('-c, --component <COMPONENT_NAME>', 'Name of the component in Storyblok space')
+  .requiredOption('-f, --field <FIELD_NAME>', 'Name of the field in this component')
+  .action(async (options) => {
+    const field = options.field || ''
+    const component = options.component || ''
+
+    const space = program.space
+    if (!space) {
+      console.log(chalk.red('X') + ' Please provide the space as argument --space YOUR_SPACE_ID.')
+      process.exit(1)
+    }
+
+    console.log(`${chalk.blue('-')} Creating the migration file to component ${component}->${field}\n`)
```

Comment:
I would write: Creating the migration file in ./migrations/change_${component}_${field}.js